plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -706,10 +706,12 @@ class FossId internal constructor(
         if (response.data?.status in SCAN_STATE_FOR_TRIGGER) {
             logger.info { "Triggering scan as it has not yet been started." }
 
-            val optionsFromConfig = arrayOf(
+            val optionsFromConfig = mutableListOf(
                 "auto_identification_detect_declaration" to "${config.detectLicenseDeclarations.compareTo(false)}",
                 "auto_identification_detect_copyright" to "${config.detectCopyrightStatements.compareTo(false)}"
-            )
+            ).apply {
```

Comment:
Didn't we say we could do the same as `auto_identification_detect_declaration` and put the default value in `FossIdConfig`?

Reply:
Yes, that's correct. However, I opted not to set the sensitivity directly to avoid potential issues if FossID changes the default value. This way, we won't need to update it throughout ORT.  
I don't know if my current implementation is too janky / doesn't meet ORT Standards.
I can change the implementation to the implementation we discussed before.

Reply:
Let's see the other people opinion then.

Reply:
This could be an interesting topic for @mnonnenmacher in the context of migrating scanners to the new plugin API.

From my POV, I'm basically fine either way, but I feel that whatever way we go for, it should be consistently implemented at least within FossID. So as the majority of properties currently comes with a default encoded, we should probably do so here, too.

Reply:
> This could be an interesting topic for @mnonnenmacher in the context of migrating scanners to the new plugin API.

I'm not sure in which way, it will stil be optional to set a default value with the new API.

> to avoid potential issues if FossID changes the default value

You could also argue that setting a default value makes the ORT implementation more stable because it would not be affected by changes in FossID itself. However, I would go with the consistency argument and just set a default like for the others.

Reply:
I have updated the code to use 10 as a default.

Reply:
> I'm not sure in which way, it will stil be optional to set a default value with the new API.

Right. I was thinking more about *where* to apply defaults, when parsing the configuration file, or as default arguments of the configuration class.